.github/workflows/test_branches.yml

```diff
@@ -521,6 +521,17 @@ jobs:
         echo ""
         echo "PYOMO_CONFIG_DIR=${GITHUB_WORKSPACE}/config" >> $GITHUB_ENV
 
+    # this has to be done after Pyomo is installed because highspy 
+    # depends on pyomo's find_library function
+    - name: Install HiGHS
+      if: matrix.PYENV == 'pip'
```

Comment:
We could put the "slim" test here in the `if:` section instead of in the action itself.  Also, does it make sense to NOT install highspy if we are running in a conda environment, or can we remove the `PYENV == 'pip'` restriction??

Reply:
Great question. Let me try.